File: app/src/main/kotlin/thanh/ha/ui/adapters/ViewPagerAdapter.java
package thanh.ha.ui.adapters;

/**
 * Created by HaVan on 5/23/2017.
 */

import androidx.fragment.app.Fragment;
import androidx.fragment.app.FragmentManager;
import androidx.fragment.app.FragmentPagerAdapter;

import java.util.List;

public class ViewPagerAdapter extends FragmentPagerAdapter {
    private List<Fragment> dataSet;

    public ViewPagerAdapter(FragmentManager manager, List<Fragment> dataSet) {
        super(manager);
        this.dataSet = dataSet;
    }


    @Override
    public Fragment getItem(int position) {
        return dataSet.get(position);
    }

    @Override
    public int getCount() {
        return dataSet.size();
    }


}

File: app/src/main/kotlin/thanh/ha/ui/adapters/ViewPagerAdapter.kt
package thanh.ha.ui.adapters

import androidx.fragment.app.Fragment
import androidx.fragment.app.FragmentManager
import androidx.fragment.app.FragmentPagerAdapter


class ViewPagerAdapter(manager: FragmentManager?, private val dataSet: List<Fragment>)
    : FragmentPagerAdapter(manager!!, BEHAVIOR_RESUME_ONLY_CURRENT_FRAGMENT) {

    override fun getItem(position: Int): Fragment {
        return dataSet[position]
    }

    override fun getCount(): Int {
        return dataSet.size
    }

}